algorithms/color_refinement.py

```diff
@@ -11,7 +11,7 @@ def color_refinement(G: "Graph"):
     """
 
```

Comment:
Ik zie niet zo goed hoe bij de normale color_refinement de G.colors weer geupdate wordt naar de nieuwe? Dus ik heb het idee als je nu de gewone color_refinement doet, dat de colors map van de graph natijd niet overeenkomt met de kleuren van de vertices van de graph

Reply:
Eigenlijk hetzelfde voor de fast_color_refinement. Heeft dat geen invloed in een volgende branching?

Reply:
De `colors` van regel 14 wordt meegegeven aan `__colorgroup_refinement`. Daar wordt op regel 104 en 105 `colors` geupdate. Omdat op regel 14 colors` wordt gelijkgesteld aan `G.colors`, wordt op regel 104 en 105 `G.colors` ook aangepast. Het is dus geen kopie.

Reply:
Ahaaaa!